ott/core/initializers.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2022 The OTT Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sinkhorn initializers."""
+from abc import ABC, abstractmethod
+from typing import Optional
+
+import jax
+import jax.numpy as jnp
+
+from ott.core import linear_problems
+from ott.geometry import pointcloud
+
+
+class SinkhornInitializer(ABC):
+
+  @abstractmethod
+  def init_dual_a(
+      self, ot_problem: linear_problems.LinearProblem, lse_mode: bool
+  ) -> jnp.ndarray:
+    """Initialization for Sinkhorn potential/ scaling f_u."""
+
+  @abstractmethod
+  def init_dual_b(
+      self, ot_problem: linear_problems.LinearProblem, lse_mode: bool
+  ) -> jnp.ndarray:
+    """Initialization for Sinkhorn potential/ scaling g_v."""
```

Comment:
Just a minor thing (sorry for being picky), but is the space after the slash in `potential/ scaling` by design? If not, I'd prefer to have it without the space, also in a few other places.

Reply:
sure

Reply:
Do you prefer "x/y" or "x / y", I think both are scattered in the existing code

Reply:
The former, thanks! (I will ensure consistency later).